lib/Validations.php

```diff
@@ -509,7 +509,7 @@ public function validates_length_of($attrs)
 				if (is_float($option))
 					throw new  ValidationsArgumentError("$range_option value cannot use a float for length.");
 
-				if (!($range_option == 'maximum' && is_null($this->model->$attribute)))
+				if (!($range_option == 'maximum' && is_null($var)))
```

Comment:
Trying very hard to not change much code from the original. There's a ton that doesn't really fit our style in this project.